azext_iot/sdk/product/aicsapi.py

```diff
@@ -0,0 +1,1033 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.service_client import SDKClient
+from msrest import Serializer, Deserializer
+from msrestazure import AzureConfiguration
+from .version import VERSION
+from msrest.pipeline import ClientRawResponse
+from msrestazure.azure_exceptions import CloudError
+import uuid
+from . import models
+
+
+class AICSAPIConfiguration(AzureConfiguration):
+    """Configuration for AICSAPI
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credentials: Credentials needed for the client to connect to Azure.
+    :type credentials: :mod:`A msrestazure Credentials
+     object<msrestazure.azure_active_directory>`
+    :param str base_url: Service URL
+    """
+
+    def __init__(
+            self, credentials, base_url=None):
+
+        if credentials is None:
+            raise ValueError("Parameter 'credentials' must not be None.")
+        if not base_url:
+            base_url = 'http://localhost'
+
+        super(AICSAPIConfiguration, self).__init__(base_url)
+
+        self.add_user_agent('aicsapi/{}'.format(VERSION))
+        self.add_user_agent('Azure-SDK-For-Python')
```

Comment:
nit: I always remove `self.add_user_agent('Azure-SDK-For-Python')` because I don't want telemetry thinking we are using official python SDK's as we manage the CLI SDK's ourselves.